integration-tests/rest-openapi/src/main/java/org/apache/camel/quarkus/component/rest/openapi/it/RestOpenApiBean.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.rest.openapi.it;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Named;
+
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@ApplicationScoped
+@Named("openapi")
+@RegisterForReflection
+public class RestOpenApiBean {
+    public String getOpenApiJson() throws IOException {
+        InputStream is = getClass().getClassLoader().getResourceAsStream("openapi.json");
+        return new String(is.readAllBytes(), StandardCharsets.UTF_8);
```

Comment:
```suggestion
        try (InputStream is = getClass().getClassLoader().getResourceAsStream("openapi.json")) {
            return new String(is.readAllBytes(), StandardCharsets.UTF_8);
        }
```

Reply:
Thanks, I've added that change here:
https://github.com/apache/camel-quarkus/pull/4125/files#diff-8d8edb85fec11f37b1364bd477d4c80e15157f5d3a4e3ae33676206b7433b931R33